packages/commitlint-config-jira/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { UserConfig } from '@commitlint/types';
+
+const config: UserConfig = {
+  extends: ['@commitlint/config-conventional'],
+  plugins: ['@bigcommerce/commitlint-plugin-jira'],
+  rules: {
+    // https://github.com/conventional-changelog/commitlint/issues/2631
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
+    'subject-case': [2, 'never', ['start-case', 'pascal-case']],
+    'subject-start-jira': [2, 'always'],
```

Comment:
Not sure it matters or not, but we could import this string from: 🤷 
```ts
import { SUBJECT_START_JIRA } from '@bigcommerce/commitlint-plugin-jira/dist/rules';
```

Reply:
It is better to avoid importing internals of a package. Either export them explicitly, or duplicate.